Resources/Prototypes/Loadouts/Jobs/Civilian/janitor.yml

```
@@ -1,3 +1,12 @@
+- type: loadoutEffectGroup
+  id: SeniorJanitorial
+  effects:
+  - !type:JobRequirementLoadoutEffect
+    requirement:
+      !type:RoleTimeRequirement
+      role: JobJanitor
+      time: 36000 #10 hrs
```

```suggestion
      time: 180000 #50 hrs
```
this is kinda a silly meme for janitor so i think a high timer is good. gotta distinguish the scrubs somehow.

---

Very good, just wanted a maintainer to give me a good amount of time. How about 52 hours tho, like 52 weeks = 1 work year? I'm going to do 52 hours.